src/Nox.Docs/Extensions/NoxSolutionEntityDomainEventsMarkdownExtensions.cs

```diff
@@ -20,9 +25,9 @@ public static IEnumerable<EntityMarkdownFile> ToMarkdownEntityDomainEvents(this
     }
 
     private static bool ShouldCreateMarkdown(Entity entity)
-        => entity.Persistence?.Create.RaiseEvents == true
-        || entity.Persistence?.Update.RaiseEvents == true
-        || entity.Persistence?.Delete.RaiseEvents == true;
+        => Events.Contains(entity.Persistence!.Create!.RaiseEvents)
```

Comment:
see entity.template comments, would make this simple

Reply:
Changes have been pushed.